docs/cloud/features/costs_savings.md

```diff
@@ -0,0 +1,60 @@
+
+Tobiko Cloud solves this problem for you. It tracks data warehouse cost estimates per model for BigQuery and Snowflake projects that use a supported pricing plan.
+
+This granular cost information allows you to directly explain/justify your cloud warehouse spend. It also uncovers the models that could most benefit from efficiency improvements.
+
+Beyond tracking your warehouse costs, Tobiko Cloud estimates how much money it saved you! Its advanced column-level impact analysis saves you even more money than open-source SQLMesh.
+
+## Supported Data Warehouse Pricing Plans
+
+Tobiko Cloud supports these cloud pricing plans:
+
+- BigQuery [On Demand](https://cloud.google.com/bigquery/pricing#on_demand_pricing)
+- Snowflake [Credits](https://docs.snowflake.com/en/user-guide/cost-understanding-compute#label-what-are-credits)
+
+## Data Warehouse Cost Configuration
+
+Configure Tobiko Cloud to show cost information by navigating to the Settings page.
+
+![Image highlighting location of the Settings link in the left site navigation](./costs_savings/costs-navigation.png)
+
+On the General settings page (1), select your pricing plan (2), enter your costs, and then save (3).
+
+![Annotated image showing locations of the general settings link, pricing plan form fields, and save button](./costs_savings/costs-steps.png)
+
+For the BigQuery On Demand plan, we supply BigQuery's default cost per terabyte processed of $6.25\*:
+![Costs form for BigQuery On-Demand](./costs_savings/costs-bigquery-on-demand.png)
+
+For the Snowflake Credits plan, we supply Snowflake's default cost per credit of $3.00\*:
+![Costs form for Snowflake Credits](./costs_savings/costs-snowflake-credits.png)
+
+If your organization has negotiated a different cost, please enter that cost into the "Cost per" field instead so your organization's data warehouse cost estimates will be more accurate.
+
+\*These are the current default costs as of January 2025, and will be updated in Tobiko Cloud as warehouse cost defaults change.
+
+## Where to find cost and savings estimates information
+
+Estimated costs and savings are displayed on the Tobiko Cloud homepage, production environment page, runs and plans pages, and individual model pages.
+
+Cost information on each page will look similar to:
+
+![Example of costs and savings data as seen on the Tobiko Cloud homepage](./costs_savings/costs-example.png)
+
+## Savings Categories
+
+When calculating your data warehouse costs, Tobiko Cloud also estimates how much money it saved you!
+
+Cost savings are broken up into three main categories:
+
+- **Prevented Reruns**: If SQLMesh has already run an execution for a change in one environment, we won't need to rerun it in another environment (such as when a model change is backfilled on a development environment and then applied to prod).
+- **Unaffected Downstream**: Because SQLMesh understands SQL, we know if a downstream model is not affected by an upstream change and can avoid re-execution.
```

Comment:
`we can detect detect when a downstream model is not affected by an upstream change and avoid an re-execution that we know would produce the same data`